nautobot/extras/models/jobs.py

```diff
@@ -334,6 +334,101 @@ def get_absolute_url(self):
         return reverse("extras:job_detail", kwargs={"slug": self.slug})
 
 
+@extras_features("graphql")
+class JobHook(OrganizationalModel):
+    """
+    A job hook defines a request that will trigger a job hook receiver when an object is created, updated, and/or
+    deleted in Nautobot. Each job hook can be limited to firing only on certain actions or certain object types.
+    """
+
+    content_types = models.ManyToManyField(
+        to=ContentType,
+        related_name="job_hooks",
+        verbose_name="Object types",
```

Comment:
We have this inconsistent across the board. For now I'll leave it since it's the same as it currently is for Webhooks, but we've got "Object types", "Object Types", "Content Type(s)", "Object(s)". If you wouldn't mind putting a 2.0 todo comment here!